### PR TITLE
Add documentation for R4 Condition diagnosis write

### PR DESCRIPTION
**Pull request by cmccormick42**
Screen Shot 2020-03-27 at 2 30 38 PM

Screen Shot 2020-03-27 at 2 30 50 PM

### CODE REVIEW

**Comment by nishanthnamana** [comment]
We should also update note for abatementDateTime, saying it is only supported when category is 'problem-list-item'.

And we need to update notes for fields like verification_status and clinical_status. As 'encounter-diagnosis' only supports certain values of verification_status and clinical_status. Basically, we need to specify 422 scenario's for these fields.

File: content/millennium/r4/general-clinical/condition.md [hunk 1]
```diff
@@ -121,6 +121,8 @@ _Implementation Notes_
 
 * Only the body fields mentioned below are supported. Unsupported fields will be ignored.
 * Modifier fields should not be provided, and will cause the transaction to fail.
+* `abatementDateTime` is not supported for Conditions with a category of `encounter-diagnosis`.
+* `encounter` is required for Conditions with a category of `encounter-diagnosis`.
```
[the remark this answers is not in the document]
Comment (cmccormick42) [reply content/millennium/r4/general-clinical/condition.md]: Moved here https://github.com/cerner/fhir.cerner.com/pull/361/commits/99e0ffc31e3ee17a661a7fd7346c5b60c2a8385f

**Comment by cmccormick42** [comment]
> We should also update note for abatementDateTime, saying it is only supported when category is 'problem-list-item'.
> 
> And we need to update notes for fields like verification_status and clinical_status. As 'encounter-diagnosis' only supports certain values of verification_status and clinical_status. Basically, we need to specify 422 scenario's for these fields.

Moved the abatement note in previous commit/comment.

Added note for clinicalStatus https://github.com/cerner/fhir.cerner.com/pull/361/commits/d8930a04f46654fdd66854e2f39e2b2c3c635413#diff-abb99d2139d79f8a2a8f1d214f5f6882R51

Aren't we allowing the same verificationStatus codes for problems and diagnoses?

**Comment by nishanthnamana** [comment]
> > We should also update note for abatementDateTime, saying it is only supported when category is 'problem-list-item'.
> > And we need to update notes for fields like verification_status and clinical_status. As 'encounter-diagnosis' only supports certain values of verification_status and clinical_status. Basically, we need to specify 422 scenario's for these fields.
> 
> Moved the abatement note in previous commit/comment.
> 
> Added note for clinicalStatus [d8930a0#diff-abb99d2139d79f8a2a8f1d214f5f6882R51](https://github.com/cerner/fhir.cerner.com/commit/d8930a04f46654fdd66854e2f39e2b2c3c635413#diff-abb99d2139d79f8a2a8f1d214f5f6882R51)
> 
> Aren't we allowing the same verificationStatus codes for problems and diagnoses?

+1, Yeah we are allowing the same verificationStatus. No need for any explicit verificationStatus notes.

**Comment by nishanthnamana** [comment]
**Validated the creates section locally**
Clinical Status
diagnosis_write_clinicalstatus

Category
diagnosis_write_category

Encounter
diagnosis_write_encounter

AbatementDateTime
diagnosis_write_abatment

**Comment by sanschutz0808** [comment]
Validated
Screen Shot 2020-04-07 at 9 29 22 AM
Screen Shot 2020-04-07 at 1 36 51 PM

**Comment by ebaranowski** [comment]
Validated:

Screen Shot 2020-04-07 at 5 24 59 PM

**Pull request merged**